R/trace_generation.R

```diff
@@ -579,11 +579,15 @@ geom2trace <- list(
       # Convert days into milliseconds
       x <- as.numeric(x) * 24 * 60 * 60 * 1000
     }
-    L <- list(x=x,
-              y=data$y,
+    # if there is more than one y-value for a particular combination of 
+    # x, PANEL, and group; then take the _max_ y.
+    data$x <- x
+    dat <- plyr::ddply(data, c("x", "PANEL", if("group"%in%names(data))"group"), 
+                       plyr::summarise, count = max(y))
+    L <- list(x=dat$x,
+              y=dat$count,
               type="bar",
               name=params$name,
-              text=data$text,
```

Comment:
Why not?  I know this is not native R/ggplot2, as it won't render in the static  `ggplot` version, but let's make use of this field for interesting hover info in the interactive `plotly` version! (Remember this email where I replied to a user with a link to your https://cpsievert.shinyapps.io/ggtree/?)